sqllineage/utils/helpers.py

```diff
@@ -5,6 +5,9 @@
 
 
 def escape_identifier_name(name: str):
+    """
+    Removes singlequotes ('), doublequotes (") and backticks (`) from a string. Used to sanitise file paths.
+    """
```

Comment:
It's not used only for paths. Mainly for table names. The comment here is inaccurate.